.github/ISSUE_TEMPLATE/BUG_BACKEND.md

```diff
@@ -0,0 +1,38 @@
+---
+name: ⚠️ Report Backend Bug
+about: Report a Backend bug on Kubeflow Pipelines
+title: "[backend] <Bug Name>"
+labels: kind/bug, area/backend
+---
+
+### Environment
+
+*  How did you deploy Kubeflow Pipelines (KFP)?
+<!-- For more information, see an overview of KFP installation options: https://www.kubeflow.org/docs/pipelines/installation/overview/. -->
+*  KFP version: 
+<!-- Specify the version of Kubeflow Pipelines that you are using. The version number appears in the left side navigation of user interface.
+To find the version number, See version number shows on bottom of KFP UI left sidenav. -->
+*  KFP SDK version: 
+<!-- Specify the output of the following shell command: $pip list | grep kfp -->
+
+
+### Steps to reproduce
+
+<!--
+Specify how to reproduce the problem. 
+This may include information such as: a description of the process, code snippets, log output, or screenshots.
+-->
+
+### Expected result
+
+<!-- What should the correct behavior be? -->
+
+### Materials and Reference
+
+<!-- Help us debug this issue by providing resources such as: sample code, background context, or links to references. -->
```

Comment:
> The Subversion project's bug reporting instructions, at http://subversion.apache.org/reporting-issues.html, are a fairly standard example of the form. Notice how they include an invitation to provide a patch to fix the bug. This is not because such an invitation will lead to a greater patch/report ratio — most users who are capable of fixing bugs already know that a patch would be welcome, and don't need to be told. The invitation's real purpose is to emphasize to all readers, especially those new to the project or new to free software in general, that the project runs on participation. In a sense, the project's current developers are no more responsible for fixing the bug than is the person who reported it. This is an important point that many new users will not be familiar with. Once they realize it, they're more likely to help make the fix happen, if not by contributing code then by providing a more thorough reproduction recipe, or by offering to test fixes that other people post. The goal is to make every user realize that there is no innate difference between himself and the people who work on the project — that it's a question of how much time and effort one puts in, not a question of who one is.

I'd prefer reducing the usage of wording `Help us`, because we do not want people to always feel we are on the hook to fix all the problems. For open source, people should expect no one is responsible, but people may offer help.

Ref: (this is a great book, I haven't finished reading all of it, but it was a good read so far) https://producingoss.com/en/managing-participants.html#users-to-participants